doc/content/getstarted.md

```diff
@@ -2,62 +2,82 @@
 title: Get Started
 ---
 
-## The Scientific Python theme for Hugo
-
 The **Scientific Python Hugo Theme** is a theme for the
 [Hugo](https://gohugo.io) static site generator based on the
-[Fresh](https://github.com/StefMa/hugo-fresh) theme.
+[PyData Sphinx Theme](https://pydata-sphinx-theme.readthedocs.io/en/latest/).
+
+To use this theme on your site, follow these instructions.
+
+## Install Hugo
 
-To use the theme, you will need to
-[download hugo](https://github.com/gohugoio/hugo/releases)
-and place it on your path.
+Hugo is the program that reads the site's source files and generates the HTML and other files that are served to visitors in their Web browsers.
```

Comment:
To use this theme, you will need Hugo. Hugo is a static site generator, i.e., it takes markdown source files and transforms them into a website.